src/vsg/state/GraphicsPipeline.cpp

```diff
@@ -107,9 +104,12 @@ GraphicsPipeline::Implementation::Implementation(Context& context, Device* devic
     _renderPass(renderPass),
     _pipelineLayout(pipelineLayout),
     _shaderStages(shaderStages),
-    _pipelineStates(pipelineStates),
+    _pipelineStates(context.defaultPipelineStates),
     _allocator(allocator)
 {
+    _pipelineStates.insert(_pipelineStates.end(), pipelineStates.begin(), pipelineStates.end());
```

Comment:
Moving the pipeline states into the GraphicsPipeline::Implemention is messier than the parent code doing the combining.  I don't know why you just didn't add to the original code.  I will merge and then revert to the original code that does the combining in a more linear step by step way rather than mixing things up in the constructor.